velox/exec/fuzzer/AggregationFuzzer.cpp

```diff
@@ -384,8 +384,16 @@ void AggregationFuzzer::go() {
 
         auto partitionKeys = generateKeys("p", argNames, argTypes);
         auto sortingKeys = generateSortingKeys("s", argNames, argTypes);
+        std::unordered_set<std::string> partitionKeySet;
+        for (auto partitionKey : partitionKeys) {
```

Comment:
Sorry for going back and forth on this. I didn't realize that generateInputDataWithRowNumber was called from multiple places. We can pass the two vectors as input and build the unordered_set within the generateInputDataWithRowNumber function. I mostly care about the data structure of partitionKeys and sortKeys to be the same.